dataflux_pytorch/lightning/dataflux_lightning_checkpoint.py

```diff
@@ -83,3 +85,26 @@ def remove_checkpoint(
 
     def teardown(self, ) -> None:
         pass
+
+
+class DatafluxLightningAsyncCheckpoint(AsyncCheckpointIO):
+    """A checkpoint manager for GCS using the :class:'AsyncCheckpointIO' interface"""
+
+    def __init__(
+        self,
+        project_name: str,
+        storage_client: Optional[storage.Client] = None,
+        enable_multipart: bool = False,
+    ):
+        super().__init__(
+            DatafluxLightningCheckpoint(project_name,
```

Comment:
enable_multipart flag no longer exists, i believe this PR may need to be synced. It's now "disable"